src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carRentCostDay = 40;
+  const isLowestDiscount = days >= 3 && days < 7;
+  const isHighestDiscount = days >= 7;
+
+  if (isLowestDiscount) {
+    return days * carRentCostDay - 20;
+  }
+
+  if (isHighestDiscount) {
+    return days * carRentCostDay - 50;
+  }
+
+  return days * carRentCostDay;
```

Comment:
You calculate `days * carRentCostDay` three times. Do it once and use a variable.
Also, numbers 20 and 50 should also be put into variables.

```suggestion
    return days * carRentCostDay - 20;
  }

  if (isHighestDiscount) {
    return days * carRentCostDay - 50;
  }

  return days * carRentCostDay;
```